service-workers/service-worker/installing.https.html

```diff
@@ -5,33 +5,44 @@
 <script src="resources/test-helpers.sub.js"></script>
 <body>
 <script>
+
+const SCRIPT = 'resources/empty-worker.js';
+const SCOPE = 'resources/blank.html';
+
 // "installing" is set
-async_test(function(t) {
-    var step = t.step_func.bind(t);
-    var url = 'resources/empty-worker.js';
-    var scope = 'resources/blank.html';
-    var frame;
+promise_test(async t => {
 
-    service_worker_unregister(t, scope)
-    .then(step(function() { return with_iframe(scope); }))
-    .then(step(function(f) {
-        frame = f;
-        return navigator.serviceWorker.register(url, {scope: scope});
-    }))
-    .then(step(function(registration) {
-        var container = frame.contentWindow.navigator.serviceWorker;
-        assert_equals(container.controller, null);
-        assert_equals(registration.active, null);
-        assert_equals(registration.waiting, null);
-        assert_equals(registration.installing.scriptURL, normalizeURL(url));
+  t.add_cleanup(async() => {
+    if(frame)
+      frame.remove();
+    if(registration)
+    await registration.unregister();
```

Comment:
indent this line so it's inside the if()

Reply:
Thanks for your correction, fixed the formatting error in newest commit.